README.md

```diff
@@ -897,9 +897,17 @@ This is an x86_64 machine
 
 #### Environment Variables
 
-- `env_var(key)` — Retrieves the environment variable with name `key`, aborting if it is not present.
+- `env_var(key)` — Retrieves the environment variable with name `key`, aborting if it is not present. It will be available as environment variable to the recipes.
+
+```make
+HELLO := env_var('HELLO')
+
+test:
+  echo "${HELLO}"
+```
+
```

Comment:
I think this section if potentially confusing. If an environment variable is set, it will be available to recipes, but you don't have to call `env_var` for that to be the case.

```suggestion
- `env_var(key)` — Retrieves the environment variable with name `key`, aborting if it is not present.
```

Reply:
Hue. Indeed. I was convinced that environment variables (from the shell) were /not/ available to recipes.
So 
`my_var := env_var('TEST')`
is just the same as 
```
my_var := `echo ${TEST}`
```
I edited the example and removed the confusing text